src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -213,6 +213,14 @@ void InputProcessor::initializeMaps()
     }
 }
 
+void InputProcessor::markObjectAsUsed(const std::string &objectType, const std::string &objectName)
+{
+    auto const find_unused = unusedInputs.find({objectType, objectName});
+    if (find_unused != unusedInputs.end()) {
+        unusedInputs.erase(find_unused);
+    }
+}
```

Comment:
:+1: :+1: :+1: This will be very handy for other work that relies on raw JSON input processing.  @mbadams5 any thoughts or concerns here?